README.md

```diff
@@ -59,136 +69,70 @@ According to original BEM-naming convention it looks like the following:
 
 *(Parameters within square brackets are optional)*
 
+**Examples:**
+
 * Block — `block-name`.
-* Block's modifier in key-value format — `block-name_mod-name_mod-val`.
-* Block's boolean modifier — `block-name_mod`.
+* Block's modifier — `block-name_mod-name_mod-val`.
+* Block's simple modifier — `block-name_mod`.
 * Block's element — `block-name__elem-name`.
-* Element's modifier in key-value format — `block-name__elem-name_mod-name_mod-val`.
-* Element's boolean modifier — `block-name__elem_mod`.
-
-Object representation of BEM entity
------------------------------------
+* Element's modifier — `block-name__elem-name_mod-name_mod-val`.
+* Element's simple modifier — `block-name__elem_mod`.
 
-BEM entities can be defined with a help of JS object with the following fields:
+### Delimiters
 
-* `block` — a block name. The field is required because only a block exists as an independent BEM entity
-* `elem` — an element name.
-* `modName` — a modifier name.
-* `modVal` — a modifier value.
+The original naming uses the following delimiters:
 
-The modifier consists of a pair of fields `modName` and `modVal`. This means that the field `modVal` without `modName` has no meaning.
+* `__` — to separate element from block
+* `_` — to separate modifier name from block or element and to separate value of modifier from name of modifier
```

Comment:
to separate a modifier name from a block or element and to separate a modifier value from a modifier name